docs/source/info.py

```diff
@@ -5,10 +5,12 @@
 # change from one version to another in conf.py should be 
 # placed here, otherwise you WILL break the build.
 
+from datetime import datetime
+
 master_doc = 'index'
 
 project = u'StackStorm'
-copyright = u'2016, StackStorm'
+copyright = u'%s, StackStorm' % (datetime.now().strftime("%Y"))
```

Comment:
Thank you for your comment. I got it.
I'll make another PR to correct this point.